modules/fedimint-dummy-server/src/lib.rs

```diff
@@ -209,24 +364,63 @@ impl ServerModule for Dummy {
 
     async fn output_status(
         &self,
-        _dbtx: &mut ModuleDatabaseTransaction<'_>,
-        _out_point: OutPoint,
+        dbtx: &mut ModuleDatabaseTransaction<'_>,
+        out_point: OutPoint,
     ) -> Option<DummyOutputOutcome> {
-        None
+        // check whether or not the output has been processed
+        dbtx.get_value(&DummyOutcomeKeyV1(out_point)).await
     }
 
-    async fn audit(&self, _dbtx: &mut ModuleDatabaseTransaction<'_>, _audit: &mut Audit) {}
+    async fn audit(&self, dbtx: &mut ModuleDatabaseTransaction<'_>, audit: &mut Audit) {
+        audit
+            .add_items(dbtx, &DummyFundsKeyV1Prefix, |k, v| match k {
+                // special account for creating assets (positive)
+                DummyFundsKeyV1(key) if key == fed_account() => v.msats as i64,
+                // a user's funds are a federation's liability (negative)
+                DummyFundsKeyV1(_) => -(v.msats as i64),
+            })
+            .await;
+    }
 
     fn api_endpoints(&self) -> Vec<ApiEndpoint<Self>> {
-        vec![api_endpoint! {
-            "dummy",
-            async |_module: &Dummy, _dbtx, _request: ()| -> () {
-                Ok(())
-            }
-        }]
+        vec![
+            api_endpoint! {
+                // API allows users ask the fed to print money
+                "print_money",
+                async |module: &Dummy, context, request: DummyPrintMoneyRequest| -> () {
+                    // TODO: A way to send messages to consensus without the DB
+                    let mut dbtx = context.dbtx();
+                    dbtx.insert_entry(&DummyPrintKeyV1(request), &()).await;
+                    module.print_notify.notify_one();
+                    Ok(())
+                }
+            },
```

Comment:
Why not just use a transaction with a special-cased input/output for that? That might also solve #2387

Reply:
Thought about it, but I wanted an example of how to use consensus items and threshold signatures instead of another transaction example.  Maybe I could demo some other way?